app.yaml.example

```diff
@@ -0,0 +1,14 @@
+  
+env: standard
+service: staging-application-monitoring-vue
+runtime: nodejs14
+
+handlers:
+  - url: /static
+    static_dir: dist/assets
+  - url: /(.*\.(json|ico|js|png|css|svg|jpg))$
+    static_files: dist/\1
+    upload: dist/.*\.(json|ico|js|png|css|svg|jpg)$
+  - url: .*
+    static_files: dist/index.html
+    upload: dist/index.html
```

Comment:
our app.yaml for our React app says 'build' everywhere instead of 'dist', which is how `vite` is naming it. So it's basically the same yaml, but some new filepaths because vite's build (dist) structure is different, and had to include css|svg|jpg in the handlers as well.